README.md

```diff
@@ -73,12 +73,15 @@ to Maven local:
 
 - `git clone https://github.com/awslabs/smithy.git`
 - `cd smithy`
-- `./gradlew publishToMavenLocal`
+- `./gradlew clean build publishToMavenLocal`
 - `cd ..`
 - `git clone https://github.com/awslabs/smithy-typescript.git`
 - `cd smithy-typescript`
 - `./gradlew build`
 
+If you're consuming smithy-typescript locally in other project, run the following command instead:
```

Comment:
```markdown
If you're consuming `smithy` or `smithy-typescript` and have already published
locally, run the following command to publish the newest contents in your local
repository:
 - `./gradlew clean publishToMavenLocal`
```

Reply:
Updated in https://github.com/trivikr/smithy-typescript/commit/730b67ed40f1ab4f6f9dec6514e4eb622d7fcd38